playbook-website/app/javascript/site_styles/_opacity_guidelines.scss

```diff
@@ -0,0 +1,48 @@
+ul {
```

Comment:
Give this a class name. This will affect EVERY list on the site. Major style bleed.

Reply:
Agreed. @cvargaspower specificity is of great importance any time you are working with CSS unless you are attempting to style everything in the browser view. This selector would potentially have unintended consequences as you can reference from the CSS courses.